internal/transformations/length.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strconv"
 )
 
-func length(data string) (string, error) {
-	return strconv.Itoa(len(data)), nil
+func length(data string) (string, bool, error) {
+	transformedData := strconv.Itoa(len(data))
+	return transformedData, data != transformedData, nil
```

Comment:
This is very interesting. I think I would return a true here always as ridicoulus cases like "1" will return a false but the intent of the transformation isn´t that cc @anuraaga

Reply:
I agree, I didn't return always true because of the corner case found of "1", but I think it is reasonable to do so

Reply:
Returning always true also is the ModSec behavior (https://github.com/SpiderLabs/ModSecurity/blob/v2/master/apache2/re_tfns.c#L556)